### DISCUSSION TITLE
input.xml of electron drag effect

### DISCUSSION THREAD

**Discussion by cumtra** [nakib/elphbolt#115] [General]
Dear Nakib,
Thank you for this very useful code.
I  am trying the example of Si considering phonon drag effect by setting the electron parameter in input.xml. But I don't know  how to set the phonon parameter to calculate the phonon drag effect.
If possible would you please share with me the imput.xml considering the electronic drag effect .

Thanks
Ran

**Comment by nakib** [comment]
Dear Ran,

Thanks for using elphbolt.

In a recent change, I seem to have removed the line `drag = .true.` in the numerics namelist. Could you please try the following?

```
&numerics
	qmesh = 6 6 6
	mesh_ref = 4 !kmesh = 24 24 24
	fsthick = 0.4 !eV about enref
	datadumpdir = './scratch/' !Or, enter suitable scratch directory
        conv_thres = 0.0001
        maxiter = 50 !Maximum number of iterations

        drag = .true.
/
```

Thanks for bringing this to my attention. I will fix it soon.

Best regards,
Nakib

**Reply by cumtra** [reply]
Dear Nakib,
Thank you for your reply！
Only the phonon drag effect can be calculated after adding drag = .true. in the example of Si.
I want to know how to consider the electron drag effect.  and I should how to set phonon in input.xml.

Thanks
Ran

**Comment by nakib** [comment]
When you turn this flag on, you will get phonon drag effect on the electron transport and electron drag effect on the phonon transport.

Best,
Nakib

**Reply by cumtra** [reply]
Thank you very much for your answer！

Best,
ran

**Comment by nakib** [comment]
Dear Ran,

Quick clarification: the flag `drag` is by default true in elphbolt. So the existing input.nml file will include the electron-phonon drag even if you don't explicitly set it to true.

Best regards,
Nakib
